Havit.Blazor.Components.Web.Bootstrap/Forms/HxInputBase.cs

```diff
@@ -145,10 +145,27 @@ public abstract class HxInputBase<TValue> : InputBase<TValue>, ICascadeEnabledCo
 	/// <summary>
 	/// Elements rendering order.
 	/// </summary>
-	protected virtual LabelValueRenderOrder RenderOrder =>
-		((this is IInputWithLabelType inputWithLabelType) && (inputWithLabelType.LabelTypeEffective == LabelType.Floating))
-		? LabelValueRenderOrder.ValueLabel
-		: LabelValueRenderOrder.LabelValue;
+	protected virtual LabelValueRenderOrder RenderOrder
+	{
+		get
+		{
+			if ((this is IInputWithLabelType inputWithLabelType) && (inputWithLabelType.LabelTypeEffective == LabelType.Floating))
+			{
+				if (this.GetType().IsGenericType && this.GetType().GetGenericTypeDefinition() == typeof(HxMultiSelect<,>))
```

Comment:
We don't want any such conditions (using reflection and concrete component types) to be present in `HxInputBase`. 
Use `override LabelValueRenderOrder => {put yout component-specific rule here}` in `HxMultiSelect` class instead.

Reply:
Got it